scripts/pre_commit_linter.py

```diff
@@ -160,6 +186,55 @@ def _lint_js_files(node_path, jscs_path, config_jscsrc, input_dir):
             num_js_files, time.time() - start_time)
 
 
+def _lint_py_files(config_pylint, input_dir):
+    """Prints a list of lint errors in changed Python files.
```

Comment:
"""Prints a list of lint errors in the given input_dir, or in all changed Python files if `input_dir` is not provided."""

(Could you similarly modify the JS message above, too?)